Composer/packages/types/src/shell.ts

```diff
@@ -147,6 +147,7 @@ export type ProjectContextApi = {
   updateDialogSchema: (_: DialogSchemaFile) => Promise<void>;
   createTrigger: (id: string, formData, autoSelected?: boolean) => void;
   createQnATrigger: (id: string) => void;
+  stopBot: (projectId?: string) => void;
```

Comment:
Can we remove the conditional projectId since we are passing currentProjectID to the function @tonyanziano ?

Reply:
sure